staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -196,7 +200,9 @@ func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (decision auth
 		if r.Status.Allowed {
 			w.responseCache.Add(string(key), r.Status, w.authorizedTTL)
 		} else {
-			w.responseCache.Add(string(key), r.Status, w.unauthorizedTTL)
+			if callerControlledAttributeSize(attr) < maxUnauthorizedCachedKeySize {
```

Comment:
why just unauthorized responses?

Reply:
I was thinking that if the user is authorized to make the request, then it is probably using real resources and it's more reasonable to cache it. I suppose if they're using real resources, this limit should never be hit, so maybe it doesn't matter? what do you think?

Reply:
OK, done.